server/block/slime_block.go

```diff
@@ -0,0 +1,45 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/entity"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// SlimeBlock is a block that slows down entities and bounces them up if they drop onto it.
+type SlimeBlock struct {
+	solid
+	transparent
+}
+
+// EntityLand ...
+func (SlimeBlock) EntityLand(_ cube.Pos, _ *world.World, e world.Entity) {
+	if e, ok := e.(fallDistanceEntity); ok {
+		e.ResetFallDistance()
+	}
+	if e, ok := e.(*entity.Item); ok {
```

Comment:
Wouldn't check for `Item` explicitly, but maybe just a general interface which allows access to Velocity and SetVelocity.

Reply:
oops i thought i reverted that

Reply:
done